openedx/features/caliper_tracking/tests/expected/edx.course.enrollment.mode_changed.json

```diff
@@ -0,0 +1,29 @@
+{
+   "@context":"http://purl.imsglobal.org/ctx/caliper/v1p1",
+   "id":"urn:uuid:921bd866-93ac-4ed1-a147-cdd1fe91e87f",
+   "type":"Enrollment",
+   "actor":{
+      "type":"Person",
+      "id":"edx.devstack.lms:18000/u/honor"
+   },
+   "action":"Used",
+   "object":{
```

Comment:
This `object` is not caliper compliant. Please read the spec and modify this accordingly.